WBooks/Extensions/Fonts.swift

```diff
@@ -22,6 +22,14 @@ extension UIFont {
         return UIFont.KailasaBold(size: 16)
     }
     
+    static var gillSansBoldTitle: UIFont {
+        return UIFont.gillSansBold(size: 20)
```

Comment:
If we have only one line, we can avoid the return 😉

Reply:
Done